src/ipc/Kid.cc

```diff
@@ -170,11 +170,21 @@ bool Kid::signaled(int sgnl) const
 }
 
 /// returns kid name
-const String& Kid::name() const
+const String& Kid::processName() const
 {
     return theName;
 }
 
+String Kid::gist() const
+{
+    // TODO: Assemble both processName() and gist() from role and ID instead.
```

Comment:
We should pass(and store) two parameters to Kid() constructor, this 'role' ('squid', 'squid-disk' or 'squid-coord'), and the corresponding index (when iterating in void Kids::init()). With this two values, we can construct gist() and processName(). Is this what required by this TODO?

Reply:
Yep.

Reply:
Done.